app/models/project.rb

```diff
@@ -1,4 +1,6 @@
 class Project < ApplicationRecord
   validates_presence_of :name, :material
   belongs_to :challenge
+  has_many :contestant_projects
+  has_many :contestants, through: :contestant_projects
```

Comment:
Nice job setting up the associations.

In here is where you would want to create a method to get the count of contestants for a project. See my comment in your `projects` controller.